hieradata_aws/production.yaml

```diff
@@ -258,3 +258,843 @@ govuk_datascrubber::ensure: 'latest'
+  ci-agent-4.ci.publishing.service.gov.uk:
+    ip: 10.1.6.24
+    host_aliases:
+      - ci-agent-4.ci
+      - ci-agent-4
+  ci-agent-3.ci.publishing.service.gov.uk:
+    ip: 10.1.6.23
+    host_aliases:
+      - ci-agent-3.ci
+      - ci-agent-3
+  ci-agent-2.ci.publishing.service.gov.uk:
+    ip: 10.1.6.22
+    host_aliases:
+      - ci-agent-2.ci
+      - ci-agent-2
+  docker-management-1.management.publishing.service.gov.uk:
+    ip: 10.3.0.80
+    host_aliases:
+      - docker-management-1.management
+      - etcd.cluster
+      - docker-management-1
+  publishing-api-3.backend.publishing.service.gov.uk:
+    ip: 10.3.3.47
+    host_aliases:
+      - publishing-api-3.backend
+      - publishing-api-3
+  publishing-api-2.backend.publishing.service.gov.uk:
+    ip: 10.3.3.46
+    host_aliases:
+      - publishing-api-2.backend
+      - publishing-api-2
+  publishing-api-1.backend.publishing.service.gov.uk:
+    ip: 10.3.3.45
+    host_aliases:
+      - publishing-api-1.backend
+      - publishing-api-1
+  docker-backend-2.backend.publishing.service.gov.uk:
+    ip: 10.3.3.112
+    host_aliases:
+      - docker-backend-2.backend
+      - docker-backend-2
+  docker-backend-1.backend.publishing.service.gov.uk:
+    ip: 10.3.3.111
+    host_aliases:
+      - docker-backend-1.backend
+      - docker-backend-1
+  rummager-elasticsearch-2.api.publishing.service.gov.uk:
+    ip: 10.3.4.56
+    host_aliases:
+      - rummager-elasticsearch-2.api
+      - rummager-elasticsearch-2
+  rummager-elasticsearch-1.api.publishing.service.gov.uk:
+    ip: 10.3.4.55
+    host_aliases:
+      - rummager-elasticsearch-1.api
+      - rummager-elasticsearch-1
+  rummager-elasticsearch-3.api.publishing.service.gov.uk:
+    ip: 10.3.4.57
+    host_aliases:
+      - rummager-elasticsearch-3.api
+      - rummager-elasticsearch-3
+  ci-agent-6.ci.publishing.service.gov.uk:
+    ip: 10.1.6.26
+    host_aliases:
+      - ci-agent-6.ci
+      - ci-agent-6
+  ci-agent-7.ci.publishing.service.gov.uk:
+    ip: 10.1.6.27
+    host_aliases:
+      - ci-agent-7.ci
+      - ci-agent-7
+  ci-agent-8.ci.publishing.service.gov.uk:
+    ip: 10.1.6.28
+    host_aliases:
+      - ci-agent-8.ci
+      - ci-agent-8
+  email-alert-api-1.backend.publishing.service.gov.uk:
+    ip: 10.3.3.40
+    host_aliases:
+      - email-alert-api-1.backend
+      - email-alert-api-1
+  email-alert-api-3.backend.publishing.service.gov.uk:
+    ip: 10.3.3.42
+    host_aliases:
+      - email-alert-api-3.backend
+      - email-alert-api-3
+  email-alert-api-2.backend.publishing.service.gov.uk:
+    ip: 10.3.3.41
+    host_aliases:
+      - email-alert-api-2.backend
+      - email-alert-api-2
+  # warehouse-postgresql-1.backend.publishing.service.gov.uk:
+  #   ip: 10.3.3.110
+  #   host_aliases:
+  #     - warehouse-postgresql-1.backend
+  #     - warehouse-postgresql-1
+  ckan-1.backend.publishing.service.gov.uk:
+    ip: 10.3.3.120
+    host_aliases:
+      - ckan-1.backend
+      - ckan-1
```

Comment:
CKAN has been flagged by PH as "migrated" (though still in development) I think it is safest to remove from here